common/persistence/cassandra/cassandraMetadataPersistenceProxy.go

```diff
@@ -53,6 +53,10 @@ func newMetadataManagerProxy(cfg config.Cassandra,
 	return &metadataManagerProxy{metadataMgr: metadataMgr, metadataMgrV2: metadataMgrV2, logger: logger}, nil
 }
 
+func (m *metadataManagerProxy) GetName() string {
```

Comment:
consider creating a baseClass that has just this method. Then all the implementations can simply inherit from this.

Reply:
Yes, I"m planning to do it in the follow up PRs for both stores and proxies.